CondCore/PhysicsToolsPlugins/plugins/DropBoxMetadata_PayloadInspector.cc

```diff
@@ -107,8 +145,12 @@ namespace {
       DBoxMetadataHelper::DBMetaDataTableDisplay theDisplay(theRecordMap);
       theDisplay.printMetaDatas();
 
-      std::string fileName(m_imageFileName);
```

Comment:
I am afraid you need to percolate this within `DBMetaDataPlotDisplay` and use it to save the result png file otherwise the PayloadInspector instance of the ConDB Browser won't be able to fetch the plot to be displayed.

Reply:
Thanks for pointing this out. I'll make the change.

Reply:
I hope this approach works.